backend/matchmaking/consumers.py

```
@@ -97,10 +97,51 @@ async def match_player(self):
+                            "white_player": white_player.username,
+                            "black_player": black_player.username,
+                            "game_id": assigned_game_id,
+                        }
+                    )
+
+                    await self.remove_player_from_queue(player_in_queue)
+
                     break
 
             if matched_user:
                 if await matched_user.has_player_been_matched():
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)